### ISSUE TITLE
onReloadContent callback does not seem to trigger when notifyPropertyChange() is called

### ISSUE THREAD

**Issue by dipeshpp** [purecloudlabs/fixtable-ember#122]
I have included `this.set('externalFilters', someValue); this.notifyPropertyChange();` in a function in the actions hash. 

I suspect I am not doing this properly. On what object should `notifyPropertyChange()` be called?

**Comment by michaelmcauley** [comment]
@dipeshpp You should call `this.notifyPropertyChange('externalFilters')`. If that doesn't work, please let me know and I'll do my best to help troubleshoot.

**Comment by dipeshpp** [comment]
Nope, that doesn't work either. Also, I couldn't find any function called `notifyPropertyChange` in the fixtable-ember source-code.

**Comment by smargovsky** [comment]
@dipeshpp [notifyPropertyChange is in Ember source](https://www.emberjs.com/api/ember/2.17/classes/Controller/methods/notifyPropertyChange?anchor=notifyPropertyChange)

Could you post your code or create a [twiddle](https://ember-twiddle.com/) so we can help further?
